site/en/blog/soft-navigations-experiment/index.md

```diff
@@ -3,12 +3,12 @@
 layout: 'layouts/blog-post.njk'
 title: Experimenting with measuring soft navigations
 description: |
- The Chrome team is working on better measuring so-called soft navigations used by Single Page Applications and a new API is now in origin trial to allow sites to experiment with this too.
+ The Chrome team is working on better measuring so-called soft navigations used by Single Page Applications and a new API is now available to allow sites to experiment with this too.
```

Comment:
"available" may confuse folks to think it shipped?

Reply:
WDYT about this then:

```suggestion
 The Chrome team is working on better measuring so-called soft navigations used by Single Page Applications and a new API is now available behind a flag to allow sites to experiment with this too.
```

Reply:
yes!